### ISSUE TITLE
Using mos for generating README?

### ISSUE THREAD

**Issue by zkochan** [bucaran/generator-rise#27]
I've created a tool called [mos](https://github.com/zkochan/mos), for generating/testing markdown files

It has plugins for generating shields, different sections of README and generating examples by combining the output with the code [mos-plugin-example](https://github.com/zkochan/mos/tree/master/plugins/mos-plugin-example)

Would you accept a pull request adding mos to this generator?

**Comment by bucaran** [comment]
@zkochan Hey Zoltan. Yeah, probably, send it and let's have a look. I would like to hear @iamstarkov's thoughts too.

**Comment by iamstarkov** [comment]
@zkochan have you considered [remark](https://github.com/wooorm/remark) to process with markdown?

**Comment by zkochan** [comment]
@iamstarkov yes. I am still thinking about whether to use a markdown parser. And if yes, which one to use. There are a lot: remark, [marked](https://github.com/chjj/marked), [markdown-it](https://github.com/markdown-it/markdown-it), [remarkable](https://github.com/jonschlinkert/remarkable)...

Although, for now, a simple regex replace does the job perfectly.

**Comment by iamstarkov** [comment]
* marked is deprecated
* markdown-it — commonmark extension
* remarkable — markdown-it grandfather, deprecated,

you also forgot commonmark, seem like a good choice, but from my experience commonmark's team is not capable of developing neither spec, neither parser. So, i'd not suggest to use markdown-it or commonmark itself.

at least for me the only suitable markdown parser is remark, (ex mdast)

**Comment by zkochan** [comment]
Thank you! It is really valuable information!

Seems like remark is pluggable, maybe it'll be possible to merge with it later

**Comment by bucaran** [comment]
@iamstarkov Thanks for sharing Vladimir.

**Comment by zkochan** [comment]
I have updated mos, now it uses remark under the hood, and can be extended with [remark-plugins](https://github.com/zkochan/mos#remark-plugins)

First I was thinking about extending remark with a `remark test` cli command, but the owner of remark thinks that it is out of scope for the project.

**Comment by bucaran** [comment]
Good for you! I am using extremely simple markdown pages these days.

Up to @iamstarkov.
